src/utils/tcr.js

```diff
@@ -0,0 +1,99 @@
+const executePending = async ({
+  batchedSend,
+  fromBlock,
+  tcrContract,
+  toBN,
+  type
+}) => {
+  const statusChangeEvents = await tcrContract.getPastEvents(
+    `${type}StatusChange`,
+    { fromBlock }
+  )
+
+  // Take most recent event for each item.
+  const latestStatusChanges = {}
+  statusChangeEvents.forEach(event => {
+    const itemKey = event.raw.topics[3]
+    if (!latestStatusChanges[itemKey]) {
+      latestStatusChanges[itemKey] = event
+      return
```

Comment:
Do you mean `continue`? Doesn't seem like you would want to return here

Reply:
Using continue would actually cause an error. With forEach iteration we use return since it is a callback function.
Snipped to test on the browser:
```
[1,2,7,8,9,20,111,900].forEach(e => {
	if (e === 8) {
		console.info('returning at ', e)
		continue // Using continue will throw an error. Swap with return and it iterates until the last item.
	}	
	console.info(e)
})
```